app/middleware/getPharmacies.js

```diff
@@ -1,21 +1,34 @@
+const request = require('request');
 const log = require('../lib/logger');
-const cache = require('memory-cache');
-const pharmacies = require('../lib/getPharmacies');
 
 function getPharmacies(req, res, next) {
   const searchPoint = res.locals.coordinates;
-  const geo = cache.get('geo');
-  const limits = { nearby: 10, open: 3 };
+  // TODO: Implement this
+  // const limits = { nearby: 10, open: 3 };
 
-  log.info('get-pharmacies-start');
-  const nearby = pharmacies.nearby(searchPoint, geo, limits);
-  log.info('get-pharmacies-end');
+  const latitude = searchPoint.latitude;
+  const longitude = searchPoint.longitude;
+
+  // TODO: Extract to ENV var
```

Comment:
Ha yeah, I thought I'd deleted that! In fact, it has been deleted in the latest version